web/components/answers/create-answer-panel.tsx

```diff
@@ -79,12 +115,21 @@ export function CreateAnswerPanel(props: { contract: FreeResponseContract }) {
         <div className="mb-1">Add your answer</div>
         <Textarea
           value={text}
-          onChange={(e) => setText(e.target.value)}
+          onChange={(e) => changeAnswer(e.target.value)}
           className="textarea textarea-bordered w-full resize-none"
           placeholder="Type your answer..."
           rows={1}
           maxLength={MAX_ANSWER_LENGTH}
         />
+        {answerError ? (
+          <AnswerError key={1} level="error" text={answerError} />
+        ) : possibleDuplicateAnswer ? (
+          <AnswerError
+            key={2}
+            level="warning"
+            text={`Did you mean "${possibleDuplicateAnswer}"?`}
```

Comment:
Maybe change to "Did you mean to bet on ..."

Reply:
Done